docs/server/requirements.txt

```diff
@@ -4,4 +4,5 @@ sphinx-rtd-theme>=0.1.9
 sphinxcontrib-napoleon>=0.4.4
 sphinxcontrib-httpdomain>=1.5.0
 pyyaml>=3.12
+aafigure>=0.6
```

Comment:
When you add something to this `requirements.txt` file, you should also add it to the `docs_require` list in `setup.py` (in the root directory of this repository).

Reply:
done.

Reply:
:+1: